makefiles/test.mk

```diff
@@ -11,8 +11,8 @@ test_unit: ## Run all unit tests
 
 .PHONY: test_e2e_evm_morse
 test_e2e_evm_morse: morse_e2e_config_warning ## Run an E2E Morse relay test
-	(cd ./e2e && TEST_PROTOCOL=morse go test -tags=e2e -count=1 -run Test_PATH_E2E_EVM)
+	DOCKER_LOG=true TEST_PROTOCOL=morse go test -v -tags=e2e -count=1 -run Test_PATH_E2E_EVM ./e2e
 
 .PHONY: test_e2e_evm_shannon
 test_e2e_evm_shannon: shannon_e2e_config_warning ## Run an E2E Shannon relay test
-	(cd ./e2e && TEST_PROTOCOL=shannon go test -tags=e2e -count=1 -run Test_PATH_E2E_EVM)
+	DOCKER_LOG=true TEST_PROTOCOL=shannon go test -v -tags=e2e -count=1 -run Test_PATH_E2E_EVM ./e2e
```

Comment:
It seems that adding `DOCKER_LOG=false` to the make target (`make test_e2e_evm_morse DOCKER_LOG=false`) does NOT override the `DOCKER_LOG=true` in the make target.

Again, I would like to voice my strong opinion that `DOCKER_LOG=true` should not be the default (especially since it cannot be overriden in the make target). We/users can easily add `DOCKER_LOG=true` to the make target if it is required but I really don't think that writing logs to a file should be the default option; IMO it's useful only when debugging but not when running standard tests.